docs/data_management/res/cheaha_project_directory.csv

```diff
@@ -0,0 +1,16 @@
+Responsibilities,PI,Manager,Members,Research Computing
+Move unused shared data to LTS/archive,Yes ✔️,Yes ✔️,Yes ✔️,
+Manage backup and recovery,Yes ✔️,Yes ✔️,Yes ✔️,
+Ensure proper usage of shared storage,Yes ✔️,Yes ✔️,Yes ✔️,
+Add/remove member to/from the project space,Yes ✔️,Yes ✔️,,Upon PI/Manager request
+Hardware capital expenses,beyond default quota,,,up to default quota
+Periodically check group membership,Yes ✔️,Yes ✔️,,
+Grant access to users to a specific folder,Yes ✔️,Yes ✔️,,
+Oversee and update access controls,Yes ✔️,Yes ✔️,,
+"Data Management and Storage: <br> Administrative (authorization and access)",Yes ✔️,,,
```

Comment:
All three of these should say "Data Security Controls" instead of "Data Management and Storage". I think that will more clearly get the message across.